marketplace-ui/src/app/shared/components/header/navigation/navigation.component.html

```diff
@@ -15,13 +15,16 @@
   <ul class="navbar-nav">
     @for (item of navItems; track $index) {
     <li class="nav-item">
-      <a [lang]="languageService.selectedLanguage()" class="nav-link" [ngClass]="
+      <a [lang]="languageService.selectedLanguage()" class="nav-link d-none d-lg-block" [ngClass]="
             item.link === '/' ? 'text-primary fw-bold active' : 'text-secondary'
           " aria-current="page" [href]="item.link">
         {{ item.label | translate }}
       </a>
+      <a [lang]="languageService.selectedLanguage()" class="nav-link d-lg-none" aria-current="page" [href]="item.link">
```

Comment:
duplicate code, please use css class to control visibility on mobile and desktop instead of duplicate code